app/models/twitter.rb

```diff
@@ -0,0 +1,54 @@
+class Twitter < ActiveRecord::Base
+  require 'rubygems'
+  require 'oauth'
+  require 'json'
+
+  $baseurl = "https://api.twitter.com"
+  $consumer_key = OAuth::Consumer.new(
+  ENV["TWITTER_CONSUMER_KEY"],
+  ENV["TWITTER_SECRET_KEY"]
+  )
+  $access_token = OAuth::Token.new(
+  ENV["TWITTER_ACCESS_TOKEN"],
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Indent the first parameter one step more than the previous line.